core/src/main/tut/README.md

```diff
@@ -14,131 +14,11 @@ Click on the demo gif below to see how PureConfig effortlessly translates your c
 
 ![](http://i.imgur.com/P6sda06.gif)
 
+## Documentation
 
-## Table of Contents
+Read the [documentation](https://pureconfig.github.io/setup-pureconfig.html).
```

Comment:
I'm not sure to understand. For what I can see, we are organized like the microsites project except that we have a tut-generated `README.md` instead of a static one because our `README.md` has an example that we want to check before publishing. If I move the `README.md` inside `docs`, nothing will work because the `tut` plugin conflicts with the `microsites` plugin and no output will be generated for the site (it's a known bug that Rob Norris discovered).

Reply:
Ok, I didn't understand the organization of the microsites project the first time I saw it. I never used microsites; from what you say it seems to expect a very particular project organization. It's better if we keep this as it is in this PR and iterate over it to improve later :)